Resources/Prototypes/_Goobstation/Balloons/balloons.yml

```
@@ -0,0 +1,376 @@
+            variation: 0.125
+            volume: -4
+      - !type:SpawnEntitiesBehavior # Balloon Loot and spawn
+         spawn:
+          SpaceCash:    
+            min: 10
+            max: 15
+          zebraBalloon:    
+            min: 2
+            max: 2
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)